gamemodes/terrortown/entities/weapons/weapon_zm_mac10.lua

```diff
@@ -28,7 +28,7 @@ SWEP.Primary.ClipSize = 30
 SWEP.Primary.ClipMax = 60
 SWEP.Primary.DefaultClip = 30
 SWEP.Primary.Automatic = true
-SWEP.Primary.Ammo = "smg1"
+SWEP.Primary.Ammo = "SMG1"
```

Comment:
why was this changed?

Reply:
because it [was wrong](https://wiki.facepunch.com/gmod/Default_Ammo_Types) and running a tolower to cover the edge case is unpleasant

Reply:
ah great catch! thanks!